examples/fluids/navierstokes.c

```diff
@@ -230,9 +230,9 @@ static PetscErrorCode CreateRestrictionFromPlex(Ceed ceed, DM dm, CeedInt P,
   ierr = DMGetLocalVector(dm, &Uloc); CHKERRQ(ierr);
   ierr = VecGetLocalSize(Uloc, &Ndof); CHKERRQ(ierr);
   ierr = DMRestoreLocalVector(dm, &Uloc); CHKERRQ(ierr);
-  CeedElemRestrictionCreate(ceed, CEED_INTERLACED, Nelem, PetscPowInt(P, dim),
-                            Ndof/fieldoff[nfields], fieldoff[nfields],
-                            CEED_MEM_HOST, CEED_COPY_VALUES, erestrict, Erestrict);
+  CeedElemRestrictionCreate(ceed, Nelem, PetscPowInt(P, dim), fieldoff[nfields],
+                            1, Ndof, CEED_MEM_HOST, CEED_COPY_VALUES, erestrict,
+                            Erestrict);
```

Comment:
How can this be enough? Don't you have to make `erestrict` be an offset? Perhaps use
```c
erestrict[eoffset++] = loc;
```
up above?

Reply:
Fixed. Yet another reason to port this to a single place.